README.md

```diff
@@ -5,6 +5,46 @@
 
 ## Quick Start (User)
 
+### Docker
+
+With Docker downloaded and installed on your system, run the following commands.
+
+## Create an .env
+
+Start up your favorite text editor and paste the uuid into the file as follows:
+
+```UUID=uuid-received-from-@AstrogatorJohn```
+
+Or you can pass it in at runtime by adding:
+
+```-e "UUID=uuid-received-from-@AstrogatorJohn"```
+
+Save the file in the `adam_home` directory as `.env`.
+
+## Pulling
+
+To pull the image from the Docker repo, run:
+
+```docker pull jfabdo/adamhome:latest```
+
+## Building
+
+If, instead of pulling a prebuilt container, you would like to build it from scratch, run the following commands:
+
+To build the container, run:
+
+```docker build --tag adamhome .```
+
+## Running
+
+If you have a `.env` file, run the following command:
+
+```docker run -e "NOTEBOOK=name-of-notebook-you-want-to-run" jfabdo/adamhome```
```

Comment:
I ran into a couple issues:

- I chose to build the Docker image, and to run the notebook, I had to remove `jfabdo/` from the tag name (so, the command I actually ran was `docker run -e "NOTEBOOK=single_run_demo.ipynb" adamhome`
- After running the docker command, I got an error

```
$ docker run -e "NOTEBOOK=single_run_demo.ipynb" adamhome
/bin/sh: 1: adamctl: not found
```

Reply:
For sure. You have to run the image as what you tag it. Once you have your own image built, you just run it as the tag.

It looks like because it's a new shell, it's not recognizing the old PATH or something. Try the code I just pushed.

Reply:
The reason I put the login in the command path was so that anyone can use their own UUID. The way it's structured now, we can't share images, because you'd be giving others your UUID

Reply:
Ok, I went back to the original design, but I added the environmental variables to /src/.bashrc, so that should fix that issue. I'm not understanding the error I'm getting, so I'll continue working on that, but if you can test it and let me know what your results are, that would be great. Thank you!

Reply:
H'okay, so... in order for people to be able to script this docker file, we have to login mid build. I'm going to test this when I have my own UUID so that I don't take up any more of your time than I have to.